core/service/src/cryptography/backup_pke.rs

```diff
@@ -21,12 +22,27 @@ struct InnerBackupPrivateKey {
     decapsulation_key: <ml_kem::kem::Kem<MlKemParams> as ml_kem::KemCore>::DecapsulationKey,
 }
 
+impl Drop for InnerBackupPrivateKey {
+    fn drop(&mut self) {
+        // Directly zeroize the underlying key bytes without creating copies
```

Comment:
I am not sure I see why this is better than just deriving `ZerorizeOnDrop`.
Can you elaborate?

Reply:
Ah, right. Of course. Sorry, was late in the day.